.scalafmt.conf

```diff
@@ -1,6 +1,7 @@
-version = 2.7.5
+version = 3.0.3
+
 maxColumn = 100
-docstrings = JavaDoc
+docstrings.style = keep
```

Comment:
It should really be `docstrings.style = Asterisk`. 
cf. https://scalameta.org/scalafmt/docs/configuration.html#docstringsstyle--asterisk
However, since the example code in Scaladoc is formatted, it is set to `docstrings.style = keep`.

<img width="1112" alt="スクリーンショット 2021-09-16 22 11 03" src="https://user-images.githubusercontent.com/884074/133618149-8f635ff5-4365-4491-bcf0-8979ada2e8c7.png">